jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -120,6 +121,9 @@ public class DiagnosticUtils {
 
 	private static final Map<String, Function<String, DiagnosticProperties>> commandTable;
 	private static final Map<String, String> helpTable;
+	static {
+		IDCacheInitializer.init();
```

Comment:
I think this could be added to the existing static initializer block (at the beginning is fine).

Reply:
thanks, moved. never thought another static block existed in the bottom!